Skyra.Tests/Core/Utils/EmojiRegexTests.cs

```diff
@@ -0,0 +1,38 @@
+using JetBrains.Annotations;
+using NUnit.Framework;
+using Skyra.Core.Utils;
+
+namespace Skyra.Tests.Core.Utils
+{
+	public class EmojiRegexTests
+	{
+		[Test]
+		public void EmojiRegex_Match([Values(
+				"😀", "😃", "😄", "😁", "😆", "😅", "🤣", "😂", "🙂", "🙃", "😉", "😊", "😇", "🥰", "😍", "🤩", "😘",
```

Comment:
might be better to put this into a const string[] at the top.

Reply:
Not possible, needs to be a constant.